docs/Manual Build Instructions.md

```diff
@@ -0,0 +1,92 @@
+
+## Manual Build Instructions for the Scan Engine and EtherCAT Custom Device for Developers and Contributors
+Follow these steps to manually build all of the components necessary to load and use the Scan Engine and EtherCAT Custom Device.
+
+### Version Management
+All projects must be open and build in the version of LabVIEW that corresponds to the VeriStand version being targeted.
+
+For example, building for VeriStand 2020 requires using LabVIEW 2020 to build each build specification.
+
+Glossary of Terms:
+- PPL = Packed Project Library designated by the file extension .lvlibp
+- LLB = LabVIEW Library designated by the file extension .llb
+
+### 1. [Scan Engine FXP Libraries](https://github.com/ni/niveristand-scan-engine-fxp-libraries)
+1. Open `FXP LLB.lvproj`
+1. Execute `Script FXP Write.vi`
+1. Execute `Script FXP Read.vi`
+1. Build the `FXP LLB` build specification for each target
+
+The final file structure of the `Built` directory should be as follows:
+- Linux_32_ARM
+    - FXP.llb
+- Linux_x64
+    - FXP.llb
+- Pharlap
+    - FXP.llb
+
+### 2. [Scan Engine Module Libraries](https://github.com/ni/niveristand-scan-engine-module-libraries)
+1. Open `Modules.lvproj`
+1. Build the `Modules` build specification for each target
+1. Build the `NI ECAT Remote IO` build specification for each target
+
+The final file structure of the `Built` directory should be as follows:
+- Linux_32_ARM
+    - Modules.lvlibp
+    - NI ECAT Remote IO.llb
+- Linux_x64
+    - Modules.lvlibp
+    - NI ECAT Remote IO.llb
+- Pharlap
+    - Modules.lvlibp
+    - NI ECAT Remote IO.llb
+
+### 3. [Scan Engine Custom Device](https://github.com/ni/niveristand-scan-engine-ethercat-custom-device)
+
+For each target type (Linux_32_ARM, Linux_x64, and Pharlap):
+
+1. Copy dependencies built from previous repositories' `Built` directory into the `Custom Device Source` directory
+    - FXP.llb
+    - Modules.lvlibp
+    - NI ECAT Remote IO.llb
+1. Open `Scan Engine.lvproj`
+    - Depending on the versions of VeriStand and LabVIEW installed on the development PC, the version of the VeriStand .NET assemblies may mismatch with the project. Use .lvproj.config files to lock the version of the assemblies used by the project, or manually navigate to the assemblies on disk at `C:\Windows\Microsoft.NET\assembly\GAC_MSIL\`.
+    - You may see dialogs concerning a mismatch of target versions when loading LLBs and PPLs for other operating systems. These are safe to ignore.
+1. Build the `Engine Release` build specification
+1. Close LabVIEW completely and repeat for the next target type
+
+For Windows using Pharlap dependencies:
+
+1. Open `Scan Engine.lvproj`
+1. Build the `Configuration Release` build specification
+
+The final file structure of the `Built` directory should be as follows (only two directory levels are shown):
+- Scan Engine
+    - data\
+    - Glyphs\
+    - Linux_32_ARM\
+    - Linux_x64\
+    - Pharlap\
+    - VxWorks\
```

Comment:
It's not clear if VxWorks should exist. I would not expect it to if #97 isn't merged.

Reply:
I removed the VxWorks section.